api/api.proto

```diff
@@ -458,6 +461,8 @@ message FastpathConfig {
   repeated FastpathReplState replicas = 4;
   // Dirty flag on volume - was attached in userspace
   bool dirty = 5;
+  // fastpath coordinator node uuid to enhance reporting
+  string coord_uuid = 6;
```

Comment:
you will need to update `SdkVersion` as well for change in api.proto

Reply:
revised, it got lost in the merge with latest, resurrected it.